tiled/src/main/kotlin/ktx/tiled/mapLayers.kt

```diff
@@ -38,3 +39,13 @@ inline fun <reified T> MapLayer.propertyOrNull(key: String): T? = properties[key
  * @return true if the property exists. Otherwise false.
  */
 fun MapLayer.containsProperty(key: String) = properties.containsKey(key)
+
+/**
+ * Returns **true** if and only if the [MapLayers] collection is empty.
+ */
+fun MapLayers.isEmpty() = this.count <= 0
```

Comment:
Is it possible for the `count` to be negative? Is that some special state that should be handled separately or just defensive programming?

Reply:
It simply returns the underlying size of the array. I now just saw that there is also a `size()` function which does exactly the same and has the same comment as `getCount()`.

Shall I convert it to == 0? I always do <= 0 because it is the exact opposite of > 0. But no, the value can never be negative in my opinion.

Reply:
@Quillraven I was just curious, I wouldn't be surprised if `-1` had some special meaning.